mln/views/api/xml/module.py

```diff
@@ -19,7 +19,8 @@ def handle_module_collect_winnings(user, request):
 	won = request.get("won") == "True"
 	# todo: arcade stats
 	if won:
-		prize = module.select_arcade_prize(user)
+		all_prizes = module.item.moduleguestyields.all()
+		prize = module._get_yield(all_prizes)

```

Comment:
The code is fine, but it's in the wrong file - this file only does XML-API specific stuff like XML parsing. This is so we can later add a JSON or whatever API and easily share code between them. Also, `_get_yield` is a private method and should only be used within `Module`. Please put this code somewhere on `Module`.

Reply:
Done